packages/apps/kadena-docs/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kadena/kadena-docs",
+  "version": "0.1.0",
+  "scripts": {
+    "_phase:build": "next build",
+    "_phase:test": "",
+    "build": "next build",
+    "cypress:open": "cypress open",
+    "dev": "next dev",
+    "lint": "next lint",
```

Comment:
Maybe we should use `"lint": "npx eslint ./src --ext .js,.ts --fix",`
https://github.com/kadena-community/kadena.js/blob/master/packages/libs/client/package.json#L17
If this doesn't work we can fix it when implementing the heft-rig for next apps